drivers/virtualbox/virtualbox.go

```diff
@@ -386,6 +376,10 @@ func (d *Driver) Start() error {
 
 	switch s {
 	case state.Stopped, state.Saved:
+		d.SSHPort, err = setPortForwarding(d.MachineName, 1, "ssh", "tcp", 22, d.SSHPort)
```

Comment:
@SvenDowideit +1 -- we have discussed some broader "health" check type things against drivers and I think it would make sense to detect these type of config issues and recover.